src/nix/hash.cc

```diff
@@ -19,47 +20,12 @@ struct CmdHashBase : Command
     FileIngestionMethod mode;
     HashFormat hashFormat = HashFormat::SRI;
     bool truncate = false;
-    HashAlgorithm ha = HashAlgorithm::SHA256;
+    HashAlgorithm hashAlgo = HashAlgorithm::SHA256;
     std::vector<std::string> paths;
     std::optional<std::string> modulus;
 
     explicit CmdHashBase(FileIngestionMethod mode) : mode(mode)
     {
-        addFlag({
-            .longName = "sri",
-            .description = "Print the hash in SRI format.",
-            .handler = {&hashFormat, HashFormat::SRI},
-        });
-
-        addFlag({
-            .longName = "base64",
-            .description = "Print the hash in base-64 format.",
-            .handler = {&hashFormat, HashFormat::Base64},
-        });
-
-        addFlag({
-            .longName = "base32",
-            .description = "Print the hash in base-32 (Nix-specific) format.",
-            .handler = {&hashFormat, HashFormat::Nix32},
-        });
-
-        addFlag({
-            .longName = "base16",
-            .description = "Print the hash in base-16 format.",
-            .handler = {&hashFormat, HashFormat::Base16},
-        });
```

Comment:
legacy flags, `--hash-format` replaces them on new command. Thus don't belong in base class.